api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -285,4 +275,17 @@ public boolean supportsPropertyName(String gpName) {
 	public List<String> findSimilarFamilyNames(String searchPhrase) {
 		return getPatientNameSearch().findSimilarFamilyNames(searchPhrase);
 	}
-}
+
+	@Override
+	public String importMpiPatient(String personId) {
```

Comment:
Okay to call this person instead of patient then.

But, are we really supposed to be fetching by an integer id from them?